### PR TITLE
Pr/svalbard flir

### PR DESCRIPTION
**Pull request by kcisek**

### CODE REVIEW

**Comment by rasmartins** [comment]
Please comment all your member variables and remove commented code.

File: src/Actuators/FLIRPTU/Task.cpp [hunk 1]
```diff
@@ -90,16 +100,23 @@ namespace Actuators
...
         m_tilt(Math::c_half_pi)
       {
+
         param("PTU Model", m_args.model)
-        .defaultValue("D48")
-        .values("D48, D300")
-        .description("Pan-Til Unit model name");
+            .defaultValue("D48")
+            .values("D48, D300")
+            .description("Pan-Til Unit model name");
```
Comment (krisklau) [inline src/Actuators/FLIRPTU/Task.cpp]: Remove these indentations.

File: src/Actuators/FLIRPTU/Task.cpp [hunk 2]
```diff
@@ -264,6 +354,8 @@ namespace Actuators
         return trimValue(pos, min_pos, max_pos);
       }
 
+
```
Comment (krisklau) [inline src/Actuators/FLIRPTU/Task.cpp]: Remove

**Comment by rasmartins** [comment]
Also take a look at the failed travis build

https://travis-ci.org/LSTS/dune/builds/121704806

**Comment by krisklau** [comment]
Yeah, this PR was not really supposed to be opened yet :) I'm helping @kcisek to resolve the issues.

**Pull request closed without merging**